packages/react-next/src/components/Pivot/Pivot.base.tsx

```diff
@@ -138,7 +139,7 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef(
           {...headerButtonProps}
           id={tabId}
           key={itemKey}
-          className={isSelected ? classNames.linkIsSelected : classNames.link}
+          className={css(classNames.link, isSelected && classNames.linkIsSelected)}
           onClick={onLinkClick.bind(this, itemKey)}
           onKeyPress={onKeyPress.bind(this, itemKey)}
           ariaLabel={link.ariaLabel}
```

Comment:
please use dashed aria props as with camel cased it is not obvious which one wins. See https://hackmd.io/@fluentui/rkMawiQdI

Reply:
since this is not convergence yet, feel free to ignore.